Exercise7/Controllers/Adapters/CountryAdapter.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections.Generic;
+using Android.Support.V7.Widget;
+using Android.Views;
+using Exercise7.Models;
+
+namespace Exercise7.Controllers.Adapters
+{
+    class CountryAdapter : RecyclerView.Adapter
+    {
+        private const int TypeNormal = 1, TypeHead = 2;
+
+        private List<Country> countries;
+
+        public CountryAdapter(List<Country> countries)
+        {
+            this.countries = countries;
+        }
+
+        public override int ItemCount => countries.Count;
```

Comment:
The number of view items should be 
```C#
countries.Count + countries.Select(x=>x.Name.First()).Distinct().Count;
```

Reply:
I don't think so. It's just 2 type of view holder : 1st type is the country name and 2nd type contain both country name and country letter so it's just countries.Count.

Reply:
Ok fine.